### PR TITLE
Add visualizations for Schema and ExampleAnomalies

### PR DESCRIPTION
**Pull request by Realsen**
This completes the work to visualize TFDV artifacts - TFMA artifacts still need to be investigated further.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/3026)

### CODE REVIEW

File: frontend/src/lib/OutputArtifactLoader.ts [hunk 1]
```diff
@@ -450,45 +495,37 @@ async function getArtifactTypes(): Promise<ArtifactType[]> {
...
+async function buildArtifactViewer(script: string[]): Promise<HTMLViewerConfig> {
+  const visualizationData: ApiVisualization = {
+    arguments: JSON.stringify({ code: script }),
+    source: '',
+    type: ApiVisualizationType.CUSTOM,
+  };
+  const visualization = await Apis.buildPythonVisualizationConfig(visualizationData);
+  if (!visualization.htmlContent) {
+    // TODO: Improve error message with details.
+    throw new Error('Failed to build TFDV artifact visualization');
```
Comment (Bobgy) [inline frontend/src/lib/OutputArtifactLoader.ts]: nit: remove TFDV?

File: frontend/src/lib/OutputArtifactLoader.ts [hunk 2]
```diff
@@ -450,45 +495,37 @@ async function getArtifactTypes(): Promise<ArtifactType[]> {
...
+    artifactTypeIds.includes(artifact.getTypeId()),
   );
 
-  const tfdvArtifactsPaths = tfdvArtifacts
-    .filter(artifact => artifact.getUri()) // uri not empty
-    .flatMap(artifact => [
-      artifact.getUri() + '/eval/stats_tfrecord', // eval uri
-      artifact.getUri() + '/train/stats_tfrecord', // train uri
-    ]);
+  const tfdvArtifactsPaths = matchingArtifacts.map(artifact => artifact.getUri()); // uri not empty
```
Comment (Bobgy) [inline frontend/src/lib/OutputArtifactLoader.ts]: To guarantee only return non empty uris, you still need the filter step.
e.g.
```
const tfdvArtifactsPaths = matchingArtifacts
  .map(artifact => artifact.getUri())
  .filter(uri => uri);
```

File: frontend/src/lib/OutputArtifactLoader.ts (hunk 1 above)
Comment (Realsen) [reply frontend/src/lib/OutputArtifactLoader.ts]: Done.

File: frontend/src/lib/OutputArtifactLoader.ts (hunk 2 above)
Comment (Realsen) [reply frontend/src/lib/OutputArtifactLoader.ts]: Done.

**Pull request merged**